content/20.treatments.md

```diff
@@ -337,8 +332,14 @@ https://www.ncbi.nlm.nih.gov/pubmed/28099856
 https://www.ncbi.nlm.nih.gov/pubmed/27911847
 -->
 The lack of benefits indicated by large studies of COVID-19 treatment with HCQ (e.g., [@doi:10/ggwzsb]) is important to recognize clinically because these medications are also associated with dangerous side effects like prolonged QT interval [@doi:10.1155/2016/4626279].
+However, [@doi:10/ggwzsb] is currently retracted after several concerns being raised.
+
 There is now evidence of increased mortality among older men receiving HCQ, and administration of HCQ and HCQ+AZ did not decrease the use of mechanical ventilation in the admitted patients [@doi:10.1101/2020.04.16.20065920]. 
 Even otherwise healthy patients had an increased risk of _de novo_ ventricular arrhythmia [@doi:10/ggwzsb].
+However, [@doi:10/ggwzsb] is retracted. 
+An importent concern not disclosed in the original publication was that whether the patients administered with HCQ/CQ 
+had similar viral load or similar symptoms to the disease as the controls.
```

Comment:
```suggestion
An important concern not disclosed in the original publication was that whether the patients administered with HCQ/CQ had similar viral load or similar symptoms as the controls.
```
Is this in reference to the retracted study? If so, I think it is a little confusing because (my understanding is) that is not why it was retracted. Perhaps you would want to reiterate why it was retracted and then say that this is also a significant concern?